main.go

```diff
@@ -125,12 +127,6 @@ func main() {
 
 	c.Log = &logger
 
-	ghClient, err = c.NewClient()
```

Comment:
I was not expecting that we need to remove lots of code from the `main.go`. 😄

Reply:
Either RunnerSet or RunnerDeployment gave me an error (forgot to copy it) so I removed them for now. Didn't feel like debugging

Reply:
I have added back the ARC code